resources/js/script.js

```diff
@@ -0,0 +1,49 @@
+$(document).ready(function () {
+    var ctx = document.getElementById('barChart');
+    const data = ctx.getAttribute('chart-data');
+    var myChart = new Chart(ctx, {
+        type: 'bar',
+        data: {
+            datasets: [{
+                label: 'Revenue statistics',
+                data: JSON.parse(data),
+                backgroundColor: [
+                    'rgba(255, 99, 132, 0.2)',
+                    'rgba(255, 159, 64, 0.2)',
+                    'rgba(255, 205, 86, 0.2)',
+                    'rgba(75, 192, 192, 0.2)',
+                    'rgba(54, 162, 235, 0.2)',
+                    'rgba(153, 102, 255, 0.2)',
+                    'rgba(201, 203, 207, 0.2)',
+                    'rgba(255, 205, 86, 0.2)',
+                    'rgba(75, 192, 192, 0.2)',
+                    'rgba(54, 162, 235, 0.2)',
+                    'rgba(153, 102, 255, 0.2)',
+                    'rgba(201, 203, 207, 0.2)'
+                ],
+                borderColor: [
+                    'rgb(255, 99, 132)',
+                    'rgb(255, 159, 64)',
+                    'rgb(255, 205, 86)',
+                    'rgb(75, 192, 192)',
+                    'rgb(54, 162, 235)',
+                    'rgb(153, 102, 255)',
+                    'rgb(201, 203, 207)',
+                    'rgb(255, 205, 86)',
+                    'rgb(75, 192, 192)',
+                    'rgb(54, 162, 235)',
+                    'rgb(153, 102, 255)',
+                    'rgb(201, 203, 207)'
+                ],
+                borderWidth: 1
+            }]
+        },
+        options: {
+            scales: {
+                y: {
+                    beginAtZero: true
+                }
```

Comment:
các options này có ý nghĩa gì em?

Reply:
options này định nghĩ trục Oy bắt đầu từ giá trị 0 ạ

Reply:
options dùng để cài đặt các tùy chọn của chartJS nhằm thiết kế biểu đồ cho phù hợp với bài toán đặt ra . Một số tùy chọn như scale, dataset, tooltip, data

Reply:
options dùng để cài đặt các tùy chọn của chartJS nhằm thiết kế biểu đồ cho phù hợp với bài toán đặt ra . Một số tùy chọn như scale, dataset, tooltip, data